src/main/kotlin/no/nav/arbeid/pam/ontologindexer/config/EnvConf.kt

```diff
@@ -0,0 +1,21 @@
+package no.nav.arbeid.pam.ontologindexer.config
+
+import org.springframework.beans.factory.annotation.Value
+import org.springframework.stereotype.Component
+
+@Component
+class EnvConf {
+
+    @Value("\${pam.stillingstitler.url:http://localhost:8189/pam-ontologi-synonymer/rest/typeahead/stilling/alle}")
```

Comment:
Synes du default-verdier er bedre enn egne properties for "dev"? Kan du si to ord om det?

Reply:
Tror jeg bare tar bort alle default verdier. Blir bare forvirrende å se hvor ting hentes fra når samme verdiene ligger i både propertyfil og som default.